File: Result_.cpp
#include <iostream>

using namespace std;

int main () {
  int number;
  cout << "Enter a number: ";
  cin >> number;
  cout << "Have Entered: " << number << endl;
  if (number >= 80) {
    cout << "You Have Earned A+";
  } 
  if (number >= 70 && number < 80) {
    cout << "You have Earned A" ;
  }

  if ( number < 33) {
    cout << "You have faield the Exam";
  }


  return 0;
}

File: result.js
let mark = prompt("Enter the Number: ");
if (mark >= 80 && mark <= 100) {
  document.write("you have earn A+");
} else if (mark >= 60 && mark <= 79) {
  document.write("you have earn A");
} else if (mark >= 40 && mark <= 59) {
  document.write("you have got B ");
} else if (mark >= 33 && mark <= 39) {
  document.write("you have passed the exam");
} else {
  document.write("you have faield ");
}